source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,284 @@
+
+   If you already have an existing text index, you need to drop the
+   index by specifying the existing text index field name with the
+   following snippet: 
+
+   .. code-block:: java
+
+      collection.dropIndex("field_name");
+   
+   If you want to delete all existing indexes except ``_id``, specify "*". 
+
+   Then, you can create a :ref:`compound index <index-compound-indexes>`. A
+   compound index is a single index that holds references to multiple
+   fields within a collection's documents. This allows you to specify
+   multiple text fields using one text index.
+
+   To enable text searches on the ``title`` and ``tag`` fields, create a
+   compound index using the following snippet:  
+
+   .. code-block:: java
+
+      collection.createIndex(Indexes.compoundIndex(Indexes.text("title"), Indexes.text("tags")));
+
+Text Search
+-----------
+
+Use the ``Filters.text()`` method to specify a text search. 
+
+The ``Filters.text()`` method uses the :doc:`Filters builder
+</fundamentals/builders/filters>` to create an instance of a :ref:`Bson
+<bson>` type. The instance is a query filter that specifies how the text
+search should operate, which you pass to the ``find()`` method. 
+
+When the ``find()`` method executes, MongoDB runs a text search on the
+fields indexed with the text index on the collection. The text search
+returns documents that contain one or more of the search terms and a
+relevance score for each result. For more information on relevance
+scores, see the :ref:`Sort by Relevance <sort_by_relevance>` section. 
+
+You can include ``TextSearchOptions`` as the second parameter of the
+``Filters.text()`` method to specify text search options such as case
+sensitivity. By default, text searches run without case sensitivity
+which means MongoDB matches documents regardless of whether the value is
+lowercase or uppercase. 
+
+To specify a case sensitive search, use the following snippet:
+
+.. code-block:: java
+
+   TextSearchOptions options = new TextSearchOptions().caseSensitive(true);
+   Bson filter = Filters.text("SomeText", options);
+
+The following sections show you how to perform the following types of
+text searches:
+
+- Search Text by a Term
+- Search Text by a Phrase
+- Search Text with Terms Excluded
+- Sort by Relevance
+
+For more information, see the following API documentation:
+
+- :java-core-api:`Filters.text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass the term as a string to the ``Filters.text()`` method to specify a
+term in your text search. 
+
+If you want to match multiple terms in your text index, separate your
+terms with spaces. When you pass the ``Filters.text()`` method to the
+``find()`` method, the text search performs a logical OR on each term
+and returns documents that contain any of the terms. 
```

Comment:
This one is tough to explain, so it might be best to introduce as few new concepts as possible. How about something like this:

```suggestion
If you want to match multiple terms in your text search, separate them
with spaces in the Filters.text builder method. When you pass the 
text search query returned by the builder method to the
``find()`` method, it returns documents that match any of the terms. 
```

Reply:
I'm removing this part and redirecting it to the Indexes page.

Reply:
I still see this section which I think belongs here. It could help to identify the material with a header, perhaps something like "Search Text by Multiple Terms"